zerver/views/registration.py

```diff
@@ -532,3 +533,33 @@ def realm_redirect(request: HttpRequest) -> HttpResponse:
         form = RealmRedirectForm()
 
     return render(request, 'zerver/realm_redirect.html', context={'form': form})
+
+@csrf_exempt
+def register_development_user(request: HttpRequest) -> HttpResponseRedirect:
+    realm = get_realm("zulip")
+    count = UserProfile.objects.count()
+    name = 'user-%d' % (count,)
+    email = '%s@zulip.com' % (name,)
+    prereg = create_preregistration_user(email, request, False)
```

Comment:
let's spell that last argument as a kwarg: `realm_creation=False, password_required=False`.